### PR TITLE
Close #1634 adds Hit and Miss tests for standalone and clustered

### LINKED ISSUE #1634: Add tests for Latency stats (standalone and clustered)
Adds tests for all tiers to test 
LatencyMinimum
LatencyMaximum
LatencyAverage

### PR DESCRIPTION
**Pull request by gibsong**

### CODE REVIEW

**Comment by gibsong** [comment]
Here is the original issue
https://github.com/ehcache/ehcache3/issues/1634

**Comment by gibsong** [comment]
test this please

**Comment by gibsong** [comment]
This was rebased today and all tests are passing.

**Comment by gibsong** [comment]
@ljacomet pretty please can you review this PR? Tests are passing, but I will probably need to rebase before a merge.

File: clustered/integration-test/src/test/java/org/ehcache/clustered/management/ClusteredStatisticsLatencyTest.java [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+    cache.get("one");   //hit
+    cache.get("two");   //hit
+
+    cache.get("three"); //miss
+    cache.get("four");  //miss
+
+
+    // it could be several seconds before the sampled stats could become available
+    // let's try until we find values that aren't defualt
```
Comment (ljacomet) [inline clustered/integration-test/src/test/java/org/ehcache/clustered/management/ClusteredStatisticsLatencyTest.java]: typo in comment

**Comment by gibsong** [comment]
test this please

**Comment by gibsong** [comment]
@ljacomet I made the typo fix, rebased and re-submitted.  Tests are passing.  Please merge when you get a chance.

**Comment by gibsong** [comment]
@mathieucarbou and @ljacomet 
I rebased and squashed the commits(sorry missed this).  Tests passed, so this should be good to merge.

**Pull request merged**